.github/workflows/ci.yaml

```diff
@@ -58,7 +58,6 @@ jobs:
         uses: actions/checkout@v3
         with:
           ref: ${{ github.event.pull_request.head.sha }}
-          fetch-depth: 20
```

Comment:
Noticed that we still had this as well, we can drop that since we no longer have the relevant CI check running that needed it